pkg/apply/applier.go

```diff
@@ -157,21 +157,30 @@ func (a *Applier) Run(ctx context.Context) <-chan event.Event {
 			// give up. Eventually we might be able to determine which errors
 			// are fatal and which might allow us to continue.
 			ch <- event.Event{
-				Type: event.ErrorEventType,
+				Type: event.ErrorType,
 				ErrorEvent: event.ErrorEvent{
 					Err: errors.WrapPrefix(err, "error applying resources", 1),
 				},
 			}
 			return
 		}
+		// If we get there, then all resources have been successfully applied.
+		ch <- event.Event{
+			Type: event.ApplyType,
+			ApplyEvent: event.ApplyEvent{
+				Type: event.ApplyEventCompleted,
```

Comment:
Shouldn't this be added to destroy as well ?

Reply:
Might be. We need to think about what events make sense to the destroyer. I will take a look at it.

Reply:
I have addressed this in a separate command in this PR. It also changes the events returned from the destroyer to be delete events rather than prune events.